packages/rpc-provider/src/http/polyfill.spec.ts

```diff
@@ -19,7 +22,7 @@ describe('http/polyfill', () => {
   });
 
   it('polyfills with no exceptions (with fetch)', () => {
-    global.fetch = () => true;
+    expect(global.fetch).toBeTruthy();
 
     expect(require('./polyfill')).toBeDefined();
   });
```

Comment:
Overall these tests were actually wrong before: `require(...)` will always be defined.

Here's the corrected version:
```typescript
// Copyright 2017-2019 @polkadot/rpc-provider authors & contributors
// This software may be modified and distributed under the terms
// of the Apache-2.0 license. See the LICENSE file for details.

import { Global } from '@polkadot/rpc-provider/mock/types';

declare const global: Global;

describe('http/polyfill', () => {
  let origFetch: GlobalFetch;

  beforeEach(() => {
    origFetch = global.fetch;
  });

  afterEach(() => {
    global.fetch = origFetch;
  });

  it('polyfills with no exceptions (without fetch)', () => {
    (global as any).fetch = undefined;
    require('./polyfill');

    expect(global.fetch).toBeDefined();
  });

  it('polyfills with no exceptions (with fetch)', () => {
    (global as any).fetch = () => true;
    require('./polyfill');

    expect(global.fetch).toBeDefined();
  });
});

```

Reply:
Ah, I'm just seeing this now! Some comments were hidden and I was busy addressing the ones on the top.
Thanks for that!